### DISCUSSION TITLE
Does the hardware buttons works as regular??

### DISCUSSION THREAD

**Discussion by alirow** [Blackymas/NSPanel_HA_Blueprint#796] [Q&A] [answered]
Does the hardware buttons works as a regular switches in v3.3? I have no values set in "Left/Right hardware button - ENTITY (Optional)" but when I click them it doesn't work. It works when I switch them from Home Assistant. They are not broken physically because click or hold custom actions defined on them works as well. 
<img width="311" alt="image" src="https://user-images.githubusercontent.com/18677903/236682735-463da309-49ba-4606-a48e-597725247253.png">
<img width="1043" alt="image" src="https://user-images.githubusercontent.com/18677903/236682779-ffa811da-b1de-461d-811e-2aa362683fa6.png">

**Comment by rleongcs** [comment, marked as answer]
The hardware buttons are currently decoupled with the relays. I believed this allow for more flexibility for people who might use/not use the relays so the hardware button can still use to control other entities. If you want to control the relay with the hardware button, then you need to put the relay entities into the hardware button entities.

![image](https://user-images.githubusercontent.com/61628312/236684279-dbcb2be1-6862-42df-a790-91fca72632a2.png)
![image](https://user-images.githubusercontent.com/61628312/236684324-b875dd9a-6af9-4a39-8b2a-049aebe37c12.png)

That's what I am doing for some of the relays that is directly connected to some lights using the very basic ESPHome config for the blueprint. Not sure if there are some direct settings in the advanced full ESPHome config.

**Reply by alirow** [reply]
Thanks for answer.
